api/schemas/report.json

```diff
@@ -114,6 +114,11 @@
             ]
         },
 
+        "master": {
+            "description": "The name of the master that was used to compile the catalog. If failover occurred, this will hold the first master successfully contacted. If this run had no master (e.g. a `puppet apply` run), this field will be blank",
```

Comment:
Will this also be populated with a master string when failover is not used? I.e, the traditional single-master workflow?